core/storage/user/gae_models.py

```diff
@@ -1629,3 +1629,37 @@ def increment_score_for_user(cls, user_id, score_category, increment_by):
         else:
             model.score += increment_by
             model.put()
+
+
+class UserCommunityRightsModel(base_models.BaseModel):
+    """Model for storing user's rights on community dashboard.
```

Comment:
Is this just for the manual phase or will it be used for automatic-granted proficiency rights as well?

If the former please add more explanation.